packages/insomnia-scripting-environment/src/objects/environments.ts

```diff
@@ -155,28 +156,28 @@ export class Vault extends Environment {
       // throw error on get or set method call if enableVaultInScripts is false
       get: (target, prop, receiver) => {
         if (!enableVaultInScripts) {
-          throw new Error('Vault is disabled in script');
+          throw toMinorScriptError(new Error('Vault is disabled in script'));
         }
         return Reflect.get(target, prop, receiver);
       },
       set: (target, prop, value, receiver) => {
         if (!enableVaultInScripts) {
-          throw new Error('Vault is disabled in script');
+          throw toMinorScriptError(new Error('Vault is disabled in script'));
         }
         return Reflect.set(target, prop, value, receiver);
       },
     });
   }
 
   unset = () => {
-    throw new Error('Vault can not be unset in script');
+    throw toMinorScriptError(new Error('Vault can not be unset in script'));
   };
 
   clear = () => {
-    throw new Error('Vault can not be cleared in script');
+    throw toMinorScriptError(new Error('Vault can not be cleared in script'));
   };
 
   set = () => {
-    throw new Error('Vault can not be set in script');
+    throw toMinorScriptError(new Error('Vault can not be set in script'));
```

Comment:
```suggestion
    throw'Vault can not be set in script';
```

If your goal is to avoid a stack trace from being included in the error.message. This approach should work without adding any new abstractions.

<img width="215" alt="image" src="https://github.com/user-attachments/assets/5c02a543-381a-4827-bcea-bffacaa4f7c9" />

Reply:
Good point, let's discuss this later.

Reply:
Updated code and description as discussed.